packages/mobile-app-did/js/pages/Token/TokenDetail/index.tsx

```diff
@@ -245,6 +257,9 @@ const TokenDetail: React.FC = () => {
             />
           )}
         </View>
+        {bannerItemsList && bannerItemsList.length > 0 && (
```

Comment:
{ bannerItemsList?.length > 0 && ... } is OK

Reply:
fixed